contributor/es/04-mechanics-of-contributing-article-es.asciidoc

```diff
@@ -0,0 +1,158 @@
+Todos ellos suelen tener en común que se basan en el texto, se archivan, se pueden buscar y vienen con enlaces estables: esto significa que tu pregunta y la respuesta quedarán escritas, y las referencias que enlaces en esas respuestas también se mantendrán accesibles.
+De este modo, podrá beneficiarse de estos conocimientos documentados de forma pasiva en su búsqueda Y ayudar a los futuros colaboradores a tener la misma ventaja.
+Esta documentación pasiva podría incluso servir para enriquecer la documentación "oficial", en caso de que contenga joyas especialmente valiosas, como definiciones importantes que se crearon ad hoc.
+
+Mientras trabajas, si encuentras documentación faltante (o desactualizada), hazle un favor al siguiente Colaborador y actualízala con lo que has descubierto.
+Los equipos de proyecto suelen estar encantados de recibir adiciones, actualizaciones o correcciones para su documentación existente: ¡acabas de encontrar otra oportunidad para contribuir!
+(O simplemente proporciónales amablemente un comentario sobre tu experiencia, y lo que te habría ayudado).
+
+==== Elaboración del código
+
+Todos tenemos nuestras preferencias y opiniones sobre el estilo del código, la sangría, etc.
+El proyecto del equipo anfitrión también las tiene.
+Trata de adaptarte y de coincidir con esas preferencias, incluso si no es lo que harías normalmente, e incluso si no está especificado en el proyecto https://innersourcecommons.org/learn/learning-path/trusted-committer/05/[_`CONTRIBUTING.md`_].
+Si no está seguro, siempre puede preguntar amablemente. No obstante, una contribución de invitado para una característica o una corrección de errores no es el momento de introducir una nueva forma de estructurar o formatear el código del proyecto.
+
+=== Enviar el pull request
+
+Has completado todo el trabajo esencial, has descubierto todas las peculiaridades del problema y del proyecto al que estás contribuyendo, se acerca el momento en el que has planeado que se utilice la nueva característica y quieres asegurarte de que tu contribución se fusione de la forma más rápida y fluida posible.
+
+Esto es lo que puedes hacer para que la revisión y la fusión sean lo más fácil posible para el https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] y el equipo anfitrión.
+Esto podría ser bastante similar a lo que ya está haciendo en su propio proyecto para conseguir que sus cambios sean aceptados. Si ese es el caso - genial, ¡esto va a ser natural para ti!
+
+==== Pruebas y automatización
+
+El punto básico aquí es permitir que el https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] valide la contribución sin tu presencia y asegurar una fácil mantenibilidad.
+Imagina que has construido una característica o el manejo de una rareza irresoluble, o un importante ajuste de rendimiento, y tu código no es del todo obvio (o incluso podría parecer hacky / incorrecto a primera vista).
```

Comment:
```suggestion
Imagina que has construido una característica o el manejo de una rareza irresoluble, o un importante ajuste de rendimiento, y tu código no es del todo obvio (o incluso podría parecer tramposo / incorrecto a primera vista).
```

Reply:
Nice catch. We indeed must avoid _hacky_.
I don't like _tramposo_, though. _truculento_, perhaps?

Reply:
Coul `lioso` fit here?

Reply:
i think truculoso fits better

Reply:
I've heard the word _enrevesado_ but never heard about the word _truculento_.
That's all I can contribute here :sweat_smile: